src/test/regress/expected/upgrade_distributed_triggers_after.out

```diff
@@ -9,9 +9,9 @@
 -- this test is relevant only for pg14-15 upgrade
 --
 SHOW server_version \gset
-SELECT substring(:'server_version', '\d+')::int = 15 AS server_version_eq_15
+SELECT substring(:'server_version', '\d+')::int = 15 AND EXISTS (SELECT * FROM pg_namespace WHERE nspname = 'upgrade_distributed_triggers') AS is_14_15_pg_upgrade
 \gset
-\if :server_version_eq_15
+\if :is_14_15_pg_upgrade
```

Comment:
could be nice to add a comment on what needs to happen when we want to switch to 15-16 upgrade

Reply:
The test description already mentions that this test is relevant only for pg14-15 upgrade, with a link to relevant PG15 commit.

Reply:
So I think it's understandable that we just don't run this test in other upgrades.

Reply:
We also have upgrade tests for PG14 straight to PG16 (without going through 15). If I understand this test its purpose correctly, then I think we should run this test for that upgrade path too.

Reply:
Not really. We generally keep two upgrade tests for latest 3 major versions. Right now, we have 13-14 and 14-15: we don't have 13-15.

Reply:
That seems like something we miss in our testing then, because skipping a version is definitely a valid upgrade path with possibly other issues than doing it step by step.

Reply:
Btw, we did have this in the past. It seems like we lost it somewhere. Let's make sure to add it back for PG16: https://github.com/citusdata/citus/blob/release-10.2/.circleci/config.yml#L710-L729

Reply:
Hmm okay then. Noted.